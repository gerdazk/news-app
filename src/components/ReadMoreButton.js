import React from "react";
import PropTypes from "prop-types";

function ReadMoreButton({ url }) {
  return (
    <a href={url} className="readMoreButton" data-testid="readMoreButton">
      Read more
    </a>
  );
}

export default ReadMoreButton;

ReadMoreButton.propTypes = {
  url: PropTypes.string,
};
