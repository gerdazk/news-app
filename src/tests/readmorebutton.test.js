import ReadMoreButton from "../components/ReadMoreButton";
import React from "react";
import { mount } from "enzyme";

describe("Read More Button component: ", () => {
  it("To match snapshot", () => {
    const wrapper = mount(
      <ReadMoreButton url="https://abcnews.go.com/Politics/biden-lay-strategy-delta-variant-covid-19/story?id=79872980" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
