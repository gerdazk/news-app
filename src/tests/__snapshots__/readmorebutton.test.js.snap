// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read More Button component:  To match snapshot 1`] = `ReactWrapper {}`;
